src/udiskssimplejob.c

```diff
@@ -109,8 +109,10 @@ udisks_simple_job_complete (UDisksSimpleJob     *job,
                             gboolean             success,
                             const gchar         *message)
 {
+  static const gchar none[] = "";
+
   g_return_if_fail (UDISKS_IS_SIMPLE_JOB (job));
-  udisks_job_emit_completed (UDISKS_JOB (job), success, message);
+  udisks_job_emit_completed (UDISKS_JOB (job), success, (message) ? message : none);
```

Comment:
@vpodzime There are subtle differences, every pointer that is initialized to the same string literal will have the same address, whereas the static constant string like I had will have a different address.  However, the nice thing is some versions of gcc will tell you if you try to `free` the static constant string at compile time.

```
warning: attempt to free a non-heap object ‘b’ [-Wfree-nonheap-object]  free((void *)b);
```
You get no such compiler warning when you attempt to free a string literal, it compiles and blows up at run-time :-)